psychopy/tools/mathtools.py

```diff
@@ -495,7 +495,7 @@ def translationMatrix(t):
     return T
 
 
-def concatenate(*args, out=None):
+def concatenate(*args, **kwargs):
```

Comment:
@mdcutone for now I'm still running the test suite on Py2 as well as Py3 so a syntax error breaks tests. I had to switch to the slightly uglier format to cope with that

Reply:
@peircej 

I think I'm going to change that function to take a list as an argument for py2 compatibility. 

I actually re-wrote all those functions as super-efficient vectorized versions, however they will yeild the same results as the current version. So we can hold off releasing that until the next point release. The functions cover 95% of what needed for 2/3D graphics and animation (point walkers, etc.) and will make working with PsychXR data much easier.